specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-05-01-preview/Incidents.json

```diff
@@ -943,6 +943,80 @@
       "type": "object",
       "x-ms-discriminator-value": "Bookmark"
     },
+    "AnomalyTimelineItem": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/EntityTimelineItem"
+        }
+      ],
+      "description": "Represents anomaly timeline item.",
+      "properties": {
+        "azureResourceId": {
+          "type": "string",
+          "description": "The anomaly azure resource id."
+        },
+        "productName": {
+          "type": "string",
+          "description": "The anomaly product name."
+        },
+        "description": {
+          "type": "string",
+          "description": "The anomaly description."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The anomaly name."
+        },
+        "endTimeUtc": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The anomaly end time."
+        },
+        "startTimeUtc": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The anomaly start time."
+        },
+        "timeGenerated": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The anomaly generated time."
+        },
+        "vendor": {
+          "type": "string",
+          "description": "The name of the anomaly vendor."
+        },
+        "intent": {
+          "type": "string",
+          "description": "The intent of the anomaly."
+        },
+        "techniques": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "technique of the anomaly"
+          },
+          "description": "The techniques of the anomaly."
+        },
+        "reasons": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Reason of the anomaly"
+          },
+          "description": "The reasons that cause the anomaly."
+        }
+      },
+      "required": [
+        "azureResourceId",
+        "displayName",
+        "endTimeUtc",
+        "startTimeUtc",
+        "timeGenerated"
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "Anomaly"
```

Comment:
Now that it defines `Anomaly`as `x-ms-discriminator-value`, `Anomaly` shall be `required`

Reply:
"Kind" is the required property and it is defined in the parent schema (EntityTimelineItem) as required. This specifies that when "Anomaly" is provided as kind, this should be the inferred type.